patientsearch/models/sync.py

```diff
@@ -87,16 +87,21 @@ def external_request(token, resource_type, params):
     if not resource_type:
         raise ValueError("Required `resource_type` not included")
 
+    user = current_user_id(token)
+    if 'DEA' not in user:
```

Comment:
Would it make sense to make this conditional on the [PDMP config being present](https://github.com/uwcirg/cosri-environments/blob/master/base/freestanding/femr/docker-compose.yaml#L17)?
```suggestion
    if current_app.config.get("EXTERNAL_FHIR_API") and 'DEA' not in user:
```

Reply:
Reasonable to wall of PDMP business rules, but this is just one of many places.  Let's take that on in a subsequent branch.